setup.py

```diff
@@ -2,21 +2,9 @@
 import sys
 
 from setuptools import setup
-from setuptools.command.test import test as TestCommand
 import versioneer
 
 
-class PyTest(TestCommand):
```

Comment:
I could not get this to work, kept getting "tests not found." I switched to a direct `pytest` call with the `pytest-cov` plugin instead, hope that is OK.